sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -91,7 +92,7 @@ export const AESTRoutesConst = {
   STUDENT_FILE_UPLOADS: Symbol(),
   STUDENT_NOTES: Symbol(),
   SIN_MANAGEMENT: Symbol(),
-  OVERAWARDS: Symbol(),
+  STUDENT_OVERAWARDS: Symbol(),
```

Comment:
Does it make sense to have the student route also renamed from `STUDENT_OVERAWARDS_BALANCE` to `STUDENT_OVERAWARDS` to have them all the same?